components/link/stories/link.stories.js

```diff
@@ -30,12 +31,14 @@ export default {
 		},
 		variant: {
 			name: "Variant",
+			defaultValue: "Primary",
```

Comment:
The case for the primary option doesn't match how it's listed in the options array.